core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -507,6 +507,28 @@ class RealityTreeReference extends RealityModelTileTree.Reference {
   }
 
   public get classifiers(): SpatialClassifiers | undefined { return undefined !== this._classifier ? this._classifier.classifiers : undefined; }
+  public createDrawArgs(context: SceneContext): TileDrawArgs | undefined {
+    // For global reality models (OSM Building layer only) - offset the reality model by the BIM elevation bias.  This would not be necessary
+    // if iModels had their elevation set correctly but unfortunately many GCS erroneously report Sea (Geoid) elevation rather than
+    // Geodetic.
+    const tree = this.treeOwner.load();
+    if (undefined === tree)
+      return undefined;
+
+    if (!this._iModel.isGeoLocated ||
+      this.computeWorldContentRange().diagonal().magnitude() < 2 * Constant.earthRadiusWGS84.equator)
+      return super.createDrawArgs(context);
+
+    const now = BeTimePoint.now();

```

Comment:
Instead of copying the superclass's implementation here, can you just call the superclass method, and then change the Z in the returned TileDrawArgs' transform?

Reply:
My concern is that future changes to the superclass method (such as I have made on my own branch) will not be reflected in the result of this method.

Reply:
Yes - fixed.